docs/OV_Converter_UG/prepare_model/convert_model/Convert_Model_From_ONNX.md

```diff
@@ -0,0 +1,55 @@
+# Converting an ONNX Model {#openvino_docs_OV_Converter_UG_prepare_model_convert_model_Convert_Model_From_ONNX}
+
+@sphinxdirective
+
+.. meta::
+   :description: Learn how to convert a model from the
+                 ONNX format to the OpenVINO Model.
+
+Introduction to ONNX
+####################
+
+`ONNX <https://github.com/onnx/onnx>`__ is a representation format for deep learning models that allows AI developers to easily transfer models between different frameworks.
+
+.. note:: ONNX model file can be loaded by ``openvino.Core.read_model`` or ``openvino.Core.compile_model`` methods by OpenVINO runtime API without preparing OpenVINO IR first. Refer to the :doc:`inference example <openvino_docs_OV_UG_Integrate_OV_with_your_application>` for more details. Using ``openvino.convert_model`` is still recommended if model load latency matters for the inference application.
+
+Converting an ONNX Model
+########################
+
+This page provides instructions on model conversion from the ONNX format to the OpenVINO IR format.
+
+Model conversion process assumes you have an ONNX model that was directly downloaded from a public repository or converted from any framework that supports exporting to the ONNX format.
+
+To convert an ONNX model, run model conversion with the path to the input model ``.onnx`` file:
+
+.. tab-set::
+
+   .. tab-item:: Python
+      :sync: py
+
+         import openvino as ov
+         ov.convert_model('your_model_file.onnx')
+
+   .. tab-item:: CLI
+      :sync: cli
+
+         ovc your_model_file.onnx
+
+External Data Files
+#########################
+
+ONNX models may consist of multiple files when the total size of the model exceeds 2GB allowed by Protobuf. According to this `ONNX article <https://github.com/onnx/onnx/blob/main/docs/ExternalData.md>`__, instead of a single file, such a model is represented as one file with ``.onnx`` extension and multiple separate files with external data which are located in the same directory where the main ``.onnx`` file is located or in another directory.
```

Comment:
```suggestion
ONNX models may consist of multiple files when the model size exceeds 2GB allowed by Protobuf. According to this `ONNX article <https://github.com/onnx/onnx/blob/main/docs/ExternalData.md>`__, instead of a single file, the model is represented as one file with ``.onnx`` extension and multiple separate files with external data. These data files are located in the same directory as the main ``.onnx`` file or in another directory.
```